<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Audio Steering</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="speedLabel">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="portComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="connectPushButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="portLabel">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="speedComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="disconnectPushButton">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Send discrete values</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="potValueSlider">
        <property name="maximum">
         <number>1023</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ringCheckBox">
        <property name="text">
         <string>Ring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="potValuePushButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>slotConnectSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnectPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>slotDisconnectSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainDialog</receiver>
   <slot>slotPortChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>potValueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>potValuePushButton</sender>
   <signal>pressed()</signal>
   <receiver>MainDialog</receiver>
   <slot>slotSetPotValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>potValuePushButton</sender>
   <signal>released()</signal>
   <receiver>MainDialog</receiver>
   <slot>slotClearPotValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotConnectSerial()</slot>
  <slot>slotDisconnectSerial()</slot>
  <slot>slotPortChanged(QString)</slot>
  <slot>slotSetPotValue()</slot>
  <slot>slotClearPotValue()</slot>
 </slots>
</ui>
